### DISCUSSION TITLE
[question] How to increase font size and ui size?

### DISCUSSION THREAD

**Discussion by CipherPower** [prasanthrangan/hyprdots#862] [Q&A]
Hey, I cant see too well and I was wondering how you make the font size larger and how to make the ui larger aswell. (like the way bar).

**Comment by kRHYME7** [comment]
for waybar 
https://github.com/prasanthrangan/hyprdots/wiki/Theming#waybar-config


Monitor: 
https://wiki.hyprland.org/Configuring/Monitors/


@CipherPower  BTW I will move this to the discussion panel.
